### ISSUE TITLE
Export dialog: problematic keyboard navigation of export range radio buttons

### ISSUE THREAD

**Issue by DavidBailes** [audacity/audacity#5454]
### Steps to reproduce

1. Ensure that the last export range used was entire project.
2. Have a project with a single audio track, and select some of the audio.
3. Open the Export Audio dialog.
4. Tab to the Export Entire project radio button.
5. Press down arrow.

### Expected behavior

The Current selection radio button is selected.

### Actual behavior

The Entire project radio button is selected, and the Edit metadata button becomes the focus.

### Audacity Version

latest beta

### Operating system

Windows

### Additional context

This issue has resurfaced since this commit: https://github.com/audacity/audacity/pull/5447
After this commit the multiple files radio button can be unavailable.

See previous relevant discussion on this issue:
https://github.com/audacity/audacity/issues/4317#issuecomment-1571712895
https://github.com/audacity/audacity/issues/4317#issuecomment-1635641928

A quick fix for this issue would be in the case that the previous export range was multiple files, and the current project doesn't have either multiple tracks or any labels, to set the Export range to Entire project, but leave the multiple files radio button available.

**Comment by DavidBailes** [comment]
> A quick fix for this issue would be in the case that the previous export range was multiple files, and the current project doesn't have either multiple tracks or any labels, to set the Export range to Entire project, but leave the multiple files radio button available.

@LWinterberg Would this fix be acceptable?

**Comment by crsib** [comment]
Ctrl->Up/Down works like a charm for me. In any case, a fix for this bug is very unlikely to make it into 3.4.0

**Comment by DavidBailes** [comment]
> Ctrl->Up/Down works like a charm for me. In any case, a fix for this bug is very unlikely to make it into 3.4.0

But Ctrl + Up/Down are not standard keystrokes for selecting radio buttons.
Why is it very unlikely to make it into 3.4.0?

**Comment by crsib** [comment]
> Why is it very unlikely to make it into 3.4.0

3.4.0 scope is now finalized unless something critical is found it will be released by EOD tomorrow.

> But Ctrl + Up/Down 

Judging by MSDN it is a standard way to move focus without selection. I don't know why wxWidgets decided to implement it differently is unknown to me, but this behavior is how it works in WX

Due to very limited scope and an easy workaround, I cannot assign this issue priority higher than P3. Unless a P1 (or a very bad P2) comes in - it won't be fixed for 3.4.0.

**Comment by crsib** [comment]
#5456 looks like P1 though

**Comment by DavidBailes** [comment]
> Judging by MSDN it is a standard way to move focus without selection.

But in a group of radio buttons, you are moving focus and selection, and your work around of ctrl+up/down arrow does that. In this case it does not move focus without selection.
Ctrl + up/down arrow are not standard keystrokes for changing the selection of a group of radio buttons.

**Comment by crsib** [comment]
I have an alternative approach, which doesn't require to enable the unusable option. Please tell me what do you think of it @DavidBailes 

https://github.com/audacity/audacity/pull/5459

**Comment by DavidBailes** [comment]
> Judging by MSDN it is a standard way to move focus without selection. I don't know why wxWidgets decided to implement it differently is unknown to me, but this behavior is how it works in WX

Deleted my first response to this, as I'd misunderstood what you were saying, and I wasn't familiar with moving focus, but not selection in groups of radio buttons.
Concerning the workaround of using Ctrl + Up/down arrow. In the context of radio buttons I doubt that may users would try this, as normally you almost always want to select, not just move focus. It's useful in a list when you want to select non contiguous items, but that isn't the case with radio buttons.
As you say, in wxWidgets, ctrl + up/down arrow in a group of radio buttons incorrectly both selects and moves focus.
